lib/open_api_spex/operation.ex

```diff
@@ -104,9 +104,11 @@ defmodule OpenApiSpex.Operation do
       [name: name, in: location, description: description, required: location == :path]
       |> Keyword.merge(opts)
 
+    schema = Keyword.get(opts, :schema, type)
```

Comment:
You can provide a schema in the `type` parameter, so we don't need a new option for this.